io.github.fragglet.sdl_sopwith.metainfo.xml

```diff
@@ -25,11 +32,12 @@
   <launchable type="desktop-id">io.github.fragglet.sdl_sopwith.desktop</launchable>
   <screenshots>
     <screenshot type="default">
-      <caption>The player dropping a bomb</caption>
       <image type="source" width="3200" height="1800">https://raw.githubusercontent.com/wjt/flathub/a70639daa7ba79dc26bc2cc7a55099c73e1fdf52/screenshots/1.png</image>
+      <caption>The player dropping a bomb</caption>
```

Comment:
TODO: Check if moving the `<caption>` element hid the false warning that it was missing; if so, file a bug on the linter.

Reply:
the check is correct, captions are not present in appstream catalogue data. compose is going wrong somewhere

![image](https://github.com/flathub/io.github.fragglet.sdl_sopwith/assets/62639087/4e4f7820-b1ea-4864-9d05-98f97db630b7)

Reply:
Removing type=source makes it work.

Reply:
Thanks. Fascinatingly, when I build with `flatpak run org.flatpak.Builder` rather than my distro's `flatpak-builder` the build fails at:

```
Finishing app
Error: Command 'sopwith' not found
```

Reply:
OK, seems like a `rofiles-fuse` bug that I am going to ignore and move on from.

Reply:
Hrm looks like it didn't work in CI...

Reply:
Yeah, and I see this same warning for other apps too, and there are essentially no `<caption>`s in the repo appstream. I think it is a global problem. I'll file a bug.

Reply:
Ah it's https://github.com/flathub-infra/website/issues/48.

Reply:
It's definitely working locally now using the build commands from https://docs.flathub.org/docs/for-app-authors/submission#before-submission, weird maybe as-glib in Flathub is somehow stripping them out.

Reply:
You're right, looks like something definitely broke on Flathub, the combined catalogue data has only 3 captions https://hub.flathub.org/repo/appstream/x86_64/appstream.xml.gz:

```
cat appstream.xml|grep "<caption>"

        <caption>Main window</caption>
        <caption>Main window</caption>
        <caption>The main user interface</caption>
```

That can't be right 🥲